packages/xo-web/src/common/utils.js

```diff
@@ -226,7 +226,7 @@ function safeHumanFormat(value, opts) {
 export const formatLogs = logs =>
   Promise.all(
     map(logs, ({ body }, id) => {
-      const matches = /^value:\s*([0-9.]+)\s+config:\s*([^]*)$/.exec(body)
+      const matches = /^value:\s*(Infinity|NaN|-Infinity|[0-9.]+)\s+config:\s*([^]*)$/.exec(body)
```

Comment:
This needs a comment explaining which values are authorized and, if available, a source for this.

Also, this likely needs to be migrated to the server side to avoid duplication of this logic.

Reply:
I'm not sure I understand what you mean by: "Also, this will probably need to be migrated to the server side to avoid duplication of this logic.".
Do you want the server parse alarms before sening it to `xo-web`?
If you are thinking about `REST API`,  a PR is already open